02_bubble/brvishnu92/main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+func TestBubble(t *testing.T) {
+
+	expected := []int{1, 2, 3, 5}
+	actual := bubble([]int{3, 2, 1, 5})
+
+	if !reflect.DeepEqual(actual, expected) {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+func TestInsertion(t *testing.T) {
+
+	expected := []int{1, 2, 3, 5}
+	actual := insertion([]int{3, 2, 1, 5})
+
+	if !reflect.DeepEqual(actual, expected) {
```

Comment:
I actually did a similar thing originally, Julia [suggested]( https://github.com/anz-bank/go-course/pull/180#discussion_r289164436) we should use testify instead.